packages/playwright-core/src/client/browserContext.ts

```diff
@@ -416,6 +416,9 @@ export async function prepareBrowserContextParams(options: BrowserContextOptions
     storageState: await prepareStorageState(options),
     serviceWorkers: options.serviceWorkers,
     recordHar: prepareRecordHarOptions(options.recordHar),
+    colorScheme: options.colorScheme ? options.colorScheme : 'no-override',
```

Comment:
This turns `undefined` into `no-override`, but it should leave our default `light` instead. We should only convert nulls into `no-override`.